src/Main.purs

```diff
@@ -211,7 +211,7 @@ execute js bundle bc@(BackendConfig backend) = do
         ]
 
       replaced = replace' (unsafeRegex """require\("[^"]*"\)""" global) (\s _ ->
-        "PS['" <> String.drop 12 (String.take (String.length s - 2) s) <> "']") (unwrap js)
+        "PS['" <> String.drop 12 (String.take (String.length s - 11) s) <> "']") (unwrap js)
```

Comment:
Bit of a nit but can we do something like `s # String.split (String.Pattern "/") # Array.index 1` here instead? That should be more robust to these sorts of changes (if this ever changes again) and it's also a little clearer, I think.